pom.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.cimpress-mcp</groupId>
+    <artifactId>sqs-streams</artifactId>
+    <version>0.6.10-SNAPSHOT</version>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>A reactive interface for consuming AWS SQS messages using rxJava, with a high-level component for queueing data to be processed by a user-supplied function with retries and exponential backoff.</description>
+    <url>https://github.com/Cimpress-MCP/sqs-streams</url>
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+    <scm>
+        <connection>scm:git:git://github.com/Cimpress-MCP/sqs-streams.git</connection>
+        <developerConnection>scm:git:ssh://github.com:Cimpress-MCP/sqs-streams.git</developerConnection>
+        <url>https://github.com/Cimpress-MCP/sqs-streams/tree/master</url>
+    </scm>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.sonatype.plugins</groupId>

```

Comment:
I followed [this guide](http://www.debonair.io/post/maven-cd/). Still need to test the publishing stuff. And need to flesh out snapshot/release mechanism